client/src/components/common/DropDown.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode } from 'react';
+import COLORS from 'src/constants/colors';
+import { fixedEntire, flexRow } from 'src/styles/common';
+import styled from 'styled-components';
+
+interface DropDownProps {
+  children: ReactNode;
+  close: () => void;
+}
+
+export default function DropDown({ children, close }: DropDownProps) {
+  const handleClick = (e: React.MouseEvent<HTMLDivElement>) => {
+    if (!(e.target as Element).className.includes('dropdown-container')) return;
+    close();
+  };
+
+  return (
+    <DropDownContainer className="dropdown-container" onClick={handleClick}>
+      <DropDownContent className="d">{children}</DropDownContent>
+    </DropDownContainer>
+  );
+}
+
+const DropDownContainer = styled.div`
+  ${fixedEntire};
+  ${flexRow};
+  z-index: 100;
```

Comment:
region 모달 처럼 background 줘볼까요? 배경 어두워 지는 것도 괜찮은 듯 해요
```suggestion
  z-index: 100;
  background: rgba(34, 34, 34, 0.3);
```

Reply:
어두워지는 것도 괜찮네요.